test/SampleCrowdsale.js

```diff
@@ -0,0 +1,95 @@
+import ether from './helpers/ether'
+import advanceToBlock from './helpers/advanceToBlock'
+import EVMThrow from './helpers/EVMThrow'
+
+const BigNumber = web3.BigNumber;
+
+const should = require('chai')
+  .use(require('chai-as-promised'))
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+const SampleCrowdsale = artifacts.require('SampleCrowdsale');
+const SampleCrowdsaleToken = artifacts.require('SampleCrowdsaleToken');
+
+contract('Crowdsale', function ([owner, wallet, investor]) {
+
+  const RATE = new BigNumber(10);
+  const GOAL = ether(10);
+  const CAP  = ether(20);
+
+  beforeEach(async function () {
+    this.startBlock = web3.eth.blockNumber + 10;
+    this.endBlock =   web3.eth.blockNumber + 20;
+
+    this.crowdsale = await SampleCrowdsale.new(this.startBlock, this.endBlock, RATE, GOAL, CAP, wallet);
+    this.token = SampleCrowdsaleToken.at(await this.crowdsale.token());
+  });
+
+  
+  it('should create crowdsale with correct parameters', async function () {
+    this.crowdsale.should.exist;
+    this.token.should.exist;
+
+    (await this.crowdsale.startBlock()).should.be.bignumber.equal(this.startBlock);
+    (await this.crowdsale.endBlock()).should.be.bignumber.equal(this.endBlock);
+    (await this.crowdsale.rate()).should.be.bignumber.equal(RATE);
+    (await this.crowdsale.wallet()).should.be.bignumber.equal(wallet);
```

Comment:
I'd have to confirm but I'm pretty sure Truffle returns `address` typed values as strings, not BigNumber instances. The test works because addresses are numbers anyway and they get converted, but it would be best to compare them as strings.